nuget/PACKAGE.md

```diff
@@ -22,7 +22,7 @@ You can call a high level `generate()` method to generate all of the output at o
 // See https://aka.ms/new-console-template for more information
 using Microsoft.ML.OnnxRuntimeGenAI;
 
-OgaHandle ogaHandle = new OgaHandle();
+using OgaHandle ogaHandle = new OgaHandle();
```

Comment:
Do we need to also provide some kind of warning to the user for the "using" keyword? @RyanUnderhill any thoughts?

Reply:
What kind of warning did you have in mind? This is a sample that gets added inside the nuget published on nuget.org: https://www.nuget.org/packages/Microsoft.ML.OnnxRuntimeGenAI#readme-body-tab

Reply:
Discussed this with Ryan, it looks good.